erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -147,18 +150,19 @@ def get_fifo_queue(filters, sle=None):
 		item_details.setdefault(key, {"details": d, "fifo_queue": []})
 		fifo_queue = item_details[key]["fifo_queue"]
 
-		transferred_item_details.setdefault((d.voucher_no, d.name), [])
+		transferred_item_key = (d.voucher_no, d.name, d.warehouse)
+		transferred_item_details.setdefault(transferred_item_key, [])
 
 		if d.voucher_type == "Stock Reconciliation":
 			d.actual_qty = flt(d.qty_after_transaction) - flt(item_details[key].get("qty_after_transaction", 0))
 
 		serial_no_list = get_serial_nos(d.serial_no) if d.serial_no else []
 
 		if d.actual_qty > 0:
-			if transferred_item_details.get((d.voucher_no, d.name)):
-				batch = transferred_item_details[(d.voucher_no, d.name)][0]
+			if transferred_item_details.get(transferred_item_key):
+				batch = transferred_item_details[transferred_item_key][0]
 				fifo_queue.append(batch)
-				transferred_item_details[((d.voucher_no, d.name))].pop(0)
+				transferred_item_details[(transferred_item_key)].pop(0)
```

Comment:
Wouldn't this work without extra parentheses?

Reply:
Fixed! thanks